.github/workflows/pre-release-pr.yml

```diff
@@ -0,0 +1,83 @@
+name: Release test pre-release version
+
+on:
+  repository_dispatch:
+    types: [pre-release-command]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.client_payload.pull_request.head.sha }}
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install deps
+        run: |
+          python -m pip install pip --upgrade
+          pip install poetry
+          pip install githubrelease
+          pip install autopub
+          pip install httpx
+      - name: Check if release exists
+        id: check_release
+        run: |
+          set +e
+          echo ::set-output name=release::$(autopub check)
+      - name: Metadata
+        id: metadata
+        if: steps.check_release.outputs.release == ''
+        run: echo "::set-output name=commit::$(git rev-parse HEAD)"
```

Comment:
Nitpick: your use of quotes around your `::set-output` echos seems inconsistent

Reply:
I guess this is what happens when you get used to having some tool fix those stuff for joy :D 😂